### DISCUSSION TITLE
Red Square on Image?

### DISCUSSION THREAD

**Discussion by beatricehc** [IBEXImagingCommunity/ibex_imaging_knowledge_base#199] [Q&A] [answered]
Hello everyone! Does anyone know why this red square appeared in my image? It is a section of the right mid-lobe of a mouse lung and was imaged according to Ce3D-IBEX protocol.
![mouselungRmid](https://github.com/user-attachments/assets/ac4ab679-f873-420d-a847-8a62618c49bd)

**Comment by zivy** [comment, marked as answer]
Hello @beatricehc,

The red square is likely due to image data corruption during data format conversion (yes, I'm stating the obvious). 

The reasoning and potential cause are [described here](https://niaid.github.io/imaris_extensions/XTRegisterSameChannel.html#results-with-artifacts). Sadly this can be an insidious type of data corruption that is hidden from the user due to the combination of an image pyramid and advanced image display (displaying the apex of the image pyramid for the low zoom level we usually use while the corruption exists at the base of the pyramid).

Potential solution - copy the original image in whatever format it was acquired to a local drive (not external), import into imaris and save to local drive (has worked for others in the past).

**Reply by beatricehc** [reply]
Great, thank you so much!

**Reply by radtkea** [reply]
Thank you @beatricehc for asking here. It's a vexing problem!!! @zivy wrote a detailed description, as you see. By asking in the open, we can relink this here. Thank you both!
